docs/source/APIs/gen

```diff
@@ -42,9 +42,7 @@ def doFile(name, dir, ent):
             line = "=" * len(title)
             fp.write(f"{title}\n{line}\n\n")
             for sym in sorted(ary):
-                fp.write(f".. autoclass:: {sym}\n" +
-                          "   :members:\n" +
-                          "   :show-inheritance:\n")
+                fp.write(f".. autoclass:: {sym}\n" + "   :members:\n" + "   :show-inheritance:\n")
```

Comment:
Does black really thing this is better?  It could at least concatenate the strings.